modules/gitops-synchronizing-cluster-configs-application.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * configuring-an-openshift-cluster-with-argo-cd.adoc
+
+[id="synchronizing-cluster-configs-application_{context}"]
+= Synchronizing `cluster configs` application
```

Comment:
Suggest `Synchronizing your application with your Git repository` do confirm with QE that this is technically right.
`cluster configs` application is something we are using for our sample flow, when you try to find information, you are likely to ask a generic query, and  `cluster configs` application at first glance might not seem relevant in a search result.